R/processing_pcs.R

```diff
@@ -3,9 +3,11 @@
 #' @param pcs The PCs to be filtered and labeled
 #' @param batches Vector of cell-wise batch information whose length is the number of
 #'   rows in `pcs`
+#' @param rename_pcs Whether to rename the PC rownames with the cell's batch label.
+#'   Default is `TRUE`.
 #'
 #' @return PCs with NA batch cells removed and labeled rownames
-filter_pcs <- function(pcs, batches) {
+  filter_pcs <- function(pcs, batches, rename_pcs = TRUE) {
```

Comment:
```suggestion
filter_pcs <- function(pcs, batches, rename_pcs = TRUE) {
```
I don't think we want the spaces here?

Reply:
no we do not, thanks!